src/adapters/hasura/altUser.adapter.ts

```diff
@@ -950,4 +950,71 @@ export class ALTHasuraUserService {
     const response = await this.axios(config);
     return response.data.data.Users.length > 0; // Returns true if username is taken
   }
+  public async getUserByToken(request: any, res: any) {
+    const authToken = request.headers.authorization;
+    const decoded: any = jwt_decode(authToken);
```

Comment:
Where do you check if this token is valid or not?

Reply:
we first decode the token, then check in the hasura if the user is present or no